src/logging/logging.bif

```diff
@@ -101,6 +101,10 @@ namespace
 
 function Log::flush_all%(%): any
 	%{
+	// Account for broker implementing its own buffering logic
+	if ( zeek::cluster::backend == zeek::broker_mgr )
+		zeek::broker_mgr->FlushLogBuffers();
```

Comment:
Mind moving this below the `log_mgr->FlushAll()`?

If we ever switch up the logic you found to also buffer writes when Broker is on, flushing the logging manager first seems more reasonable.

Otherwise LGTM, seems a fair stop-gap.